components/Funders.vue

```diff
@@ -1,20 +1,20 @@
 <template>
 	<div class="funders">
-		<h2>Funded by</h2>
-		<ul>
-			<li v-for="f in funders" :key="f.id">
-				<nuxt-link :to="f.href">
-					<img :src="f.logo" :alt="f.logoAlt">
-					{{ f.title }}
-				</nuxt-link>
-			</li>
-		</ul>
+		<p>Supported by Innovation, Science and Economic Development Canada’s Accessible Technology Program.</p>
+		<div>
+			<CanadaLogo />
```

Comment:
Should this be linked to ISED (Accessible Technology Program)? Regardless, it needs an accessible label.